src/GeneralStateTestsFiller/stEWASMTests/getExternalCodeSizeFiller.yml

```diff
@@ -35,7 +35,7 @@ getExternalCodeSize:
           (export "main" (func $main))
 
           (func $main
-            ;; for now just use our own address since it's easier
+            ;; check this account codesize
```

Comment:
I think this still uses the `0xdeadbeef` account as that's the one in the data section.

Reply:
Yes it is still using `0xdeadbeef` here https://github.com/ewasm/tests/blob/b856f9eb2c9589f703e5bc1e7b9376f2faaaa203/src/GeneralStateTestsFiller/stEWASMTests/getExternalCodeSizeFiller.yml#L43-L48 , but it also check its own code size here: https://github.com/ewasm/tests/blob/b856f9eb2c9589f703e5bc1e7b9376f2faaaa203/src/GeneralStateTestsFiller/stEWASMTests/getExternalCodeSizeFiller.yml#L38-L41